dataset/balance_data.py

```diff
@@ -0,0 +1,83 @@
+import h5py
+import pandas as pd
+from utils.helpers import *
+
+def undersample(df):
+    """Undersamples data as per particle class 
+
+    Args:
+        df (pandas dataframe): Dataframe containing entries for a momentum bin
+
+    Returns:
+        pandas dataframe: Dataframe after undersampling entries
+    """
+    classes = df.label.value_counts().to_dict()
```

Comment:
We need to comment this code so that TRIUMF knows what each part of code is doing